apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/expectExtensions.kt

```diff
@@ -15,19 +16,37 @@ import ch.tutteli.atrium.reporting.reporter
 annotation class ExperimentalWithOptions
 
 /**
- * Uses the given [textRepresentation] as representation of the subject instead of the current representation
- * (which defaults to the subject itself).
+ * Wraps the given [textRepresentation] into a [RawString] and uses it as representation of the subject
+ * instead of the so far defined representation (which defaults to the subject as such).
+ *
+ * In case [Expect.maybeSubject] is not defined i.e. [None], then the previous representation is used.
  */
 @ExperimentalWithOptions
-fun <T> RootExpect<T>.withOptions(textRepresentation: String): Expect<T> =
-    withOptions(ExpectOptions(representation = RawString.create(textRepresentation)))
+fun <T> RootExpect<T>.withRepresentation(textRepresentation: String): Expect<T> =
```

Comment:
why the upcast? I.e. why does this method not return `RootExpect<T>`?

Reply:
because:
- we don't restrict ourselfs and we can return a different representation later on without binary bc; I am not yet sure if we can return RootExpect once we have an immutable structure
- IMO it does not make sense to do `withOptions(...).withOptions(...)` one can set a property multiple times within the withOptions lambda.

The same reasoning applies to FeatureExpect below